test/legacy_test/test_imperative_base.py

```diff
@@ -14,15 +14,15 @@
 
 import contextlib
 
-from paddle import base
+from paddle import static
 
 
 @contextlib.contextmanager
 def new_program_scope(main=None, startup=None, scope=None):
-    prog = main if main else base.Program()
-    startup_prog = startup if startup else base.Program()
-    scope = scope if scope else base.core.Scope()
-    with base.scope_guard(scope):
-        with base.program_guard(prog, startup_prog):
-            with base.unique_name.guard():
+    prog = main if main else static.Program()
+    startup_prog = startup if startup else static.Program()
+    scope = scope if scope else static.core.Scope()
+    with static.scope_guard(scope):
+        with static.program_guard(prog, startup_prog):
+            with static.unique_name.guard():
```

Comment:
不需要所有 base 都改，除 Program 和 program_guard 外的都应该需要复原，否则会导致其他的单测没法通过

```suggestion
    prog = main if main else static.Program()
    startup_prog = startup if startup else static.Program()
    scope = scope if scope else base.core.Scope()
    with static.scope_guard(scope):
        with static.program_guard(prog, startup_prog):
            with base.unique_name.guard():
```